docs: update language for custom protocol service timelines (#20606)
OpenTrons/opentrons-api: docs/flex/docs/protocols/development-service.md

title: "Opentrons Flex: Custom Protocol Service"
---

> Opentrons provides a [Remote Custom Protocol Development service](https://opentrons.com/instrument-services) for applications not already included in the Protocol Library. As part of the service, Opentrons field applications scientists will:

Reviewer: This service doesn't really have a formal name, so I wonder about the capitalization. I'd make this lower case. Also, small text suggestion in the sentence:

```suggestion
Opentrons provides [custom protocol development services](https://opentrons.com/instrument-services) for applications that are not included in the Protocol Library. As part of the service, Opentrons field applications scientists will:
```

You could also write, "for applications not found in the Protocol Library."
